content/swagger/resources/reference_entity_attributes/definitions/reference_entity_attribute.yaml

```diff
@@ -50,9 +50,9 @@ properties:
   validation_rule:
     type: string
     description: Validation rule type used to validate the attribute value when the attribute type is `text`
-    default : null
+    default : "none"
     x-warning: Only for `text` attribute type
-    enum: ['email', 'url', 'regexp', null]
+    enum: ['email', 'url', 'regexp', 'none']
```

Comment:
it means that it does no accept null? Should it be mandatory then, with the API?

Reply:
I doesn't accept null, but 'none' is the default value if the property is not defined in the request

Reply:
Yes, but so is it optional or mandatory?